src/ngraph/pass/visualize_tree.hpp

```diff
@@ -37,6 +41,13 @@ class ngraph::pass::VisualizeTree : public ModulePass
     bool run_on_module(std::vector<std::shared_ptr<ngraph::Function>>&) override;
 
     static std::string get_file_ext();
+    void set_ops_to_details(
+        std::unordered_map<std::type_index,
```

Comment:
As much as I hate `using ops_to_detail_t = std::unordered_map<stuff>` I think I might hate seeing this line repeated over and over.